paywall/src/stories/content/CheckoutContent.stories.js

```diff
@@ -120,151 +120,160 @@ storiesOf('Checkout page', module)
   ))
   .add('Checkout page', () => {
     // set the data needed to display the checkout
-    useEffect(() => {
-      const messageTemplate = {
-        type: 'message',
-        source: fakeWindow.parent,
-        origin: 'origin',
-      }
-      fakeWindow.handlers.message.forEach(postedMessage => {
-        postedMessage({
-          ...messageTemplate,
-          data: {
-            type: POST_MESSAGE_CONFIG,
-            payload: paywallConfig,
-          },
-        })
-        setTimeout(() => {
-          postedMessage({
-            ...messageTemplate,
-            data: {
-              type: POST_MESSAGE_UPDATE_ACCOUNT,
-              payload: lockAddress1,
-            },
-          })
+    const Wrapper = () => {
```

Comment:
The Git diff on this file is getting computed really weirdly. The only change is that each of these stories is put into a wrapper function. This solves an error caused by using the `useEffect` hook outside a component, which only surfaced after removing redux.